docs/examples/extensions/add-task/woocommerce-admin-add-task-example.php

```diff
@@ -41,3 +41,12 @@ function add_task_register_script() {
 	do_action( 'add_woocommerce_extended_task_list_item', 'woocommerce_admin_add_task_example_name' );
 }
 add_action( 'admin_enqueue_scripts', 'add_task_register_script' );
+
+/**
+ * Unregister task list item.
+ */
+function pluginprefix_deactivate() {
+	do_action( 'remove_woocommerce_extended_task_list_item', 'woocommerce_admin_add_task_example_name' );
```

Comment:
This API is in reverse. Our plugin should fire the `do_action()` or `apply_filters()` and other plugins looking to make use of those hook should attach with `add_action()` or `add_filter()`.